### PR TITLE
added PHP 8.1 as experimental

### LINKED ISSUE #3494: PHP 8.1 - realpath(): Passing null to parameter #1 ($path) of type string is deprecated
https://github.com/phpmyadmin/phpmyadmin/pull/16743/checks?check_run_id=2156536145#step:10:8
```
PHP Deprecated:  realpath(): Passing null to parameter #1 ($path) of type string is deprecated in vendor/twig/twig/src/Loader/FilesystemLoader.php on line 40
```

### PR DESCRIPTION
**Pull request by oleg-andreyev**
fixes #3494

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -51,7 +55,9 @@ jobs:
                   key: ${{ runner.os }}-${{ matrix.php-version }}-composer-${{ hashFiles('composer.json') }}
                   restore-keys: ${{ runner.os }}-${{ matrix.php-version }}-composer-
 
-            - run: composer install
+            - run: |
+                composer --version
+                composer install ${{ matrix.composer-options }}
```
Comment (williamdes) [inline .github/workflows/ci.yml]:
```suggestion
            - run: composer install ${{ matrix.composer-options }}
```

**Review by williamdes — APPROVED** [review]

**Comment by fabpot** [comment]
This should probably be done on the 1.x branch first.

**Review by williamdes — APPROVED** [review]
💯

**Comment by fabpot** [comment]
Thank you @oleg-andreyev.

**Pull request merged**